<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInsert">
       <attribute name="title">
        <string>随机插入</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labInsert">
          <property name="text">
           <string>请在下方左侧输入合法的Json对象：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labInsertInput">
              <property name="text">
               <string>输入：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditInsertInput"/>
            </item>
            <item>
             <widget class="QLabel" name="labInsertInputDisplay">
              <property name="text">
               <string>格式化输入：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditInsertInputDisplay"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labInsertOutput">
              <property name="text">
               <string>输出：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditInsertOutput"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnInsert">
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCompare">
       <attribute name="title">
        <string>比较Json</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labCompare">
          <property name="text">
           <string>请在下方输入需要比较的两个Json字符串：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="labCompareInput1">
              <property name="text">
               <string>输入1：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditCompareInput1"/>
            </item>
            <item>
             <widget class="QLabel" name="labCompareInputDisplay1">
              <property name="text">
               <string>格式化输入1：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditCompareInputDisplay1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="labCompareInput2">
              <property name="text">
               <string>输入2：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditCompareInput2"/>
            </item>
            <item>
             <widget class="QLabel" name="labCompareInputDisplay2">
              <property name="text">
               <string>格式化输入2：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txtEditCompareInputDisplay2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnCompare">
          <property name="text">
           <string>比较</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labHint">
        <property name="text">
         <string>提示：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditHint"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txtEditInsertInput</tabstop>
  <tabstop>txtEditInsertOutput</tabstop>
  <tabstop>txtEditInsertInputDisplay</tabstop>
  <tabstop>btnInsert</tabstop>
  <tabstop>lineEditHint</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>txtEditCompareInput1</tabstop>
  <tabstop>txtEditCompareInputDisplay1</tabstop>
  <tabstop>txtEditCompareInput2</tabstop>
  <tabstop>txtEditCompareInputDisplay2</tabstop>
  <tabstop>btnCompare</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
